src/include/sof/audio/kpb.h

```diff
@@ -41,10 +41,9 @@ struct comp_buffer;
  * the sof-dosc in [kpbm-state-diagram]
```

Comment:
@marc-hb I would rather add the git commit id inside commit message because it would be one git log away from the reader.

Also, use the Fixes tag.

e.g Fixes abcdef ("Commit subject of abdef")

Reply:
Done